### PR TITLE
Make comments flat again #798

### PR DESCRIPTION
**Pull request by umgelurgel**
Removed most of the complexity out of hcomments, removed unused views, templates, forms and models and unnecessary customization of the `django_comments.comment` model. Added tests of the changes.

All `hcomments` templates were removed as they were duplicates of the files in `template/hcomments`.

### CODE REVIEW

**Comment by artcz** [comment]
@umgelurgel there are some conflicts due to python 3 migration.  could you please take a look at those?

**Comment by umgelurgel** [comment]
Rebased and working under Python3 :)

File: conference/tests/factories/talk.py [hunk 1]
```diff
@@ -3,20 +3,21 @@
...
 
-fake = Faker()
 
 class TalkFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = 'conference.Talk'
 
-    title = factory.LazyAttribute(lambda talk: fake.sentence(nb_words=6, variable_nb_words=True)[:80])
+    title = factory.LazyAttribute(
+        lambda talk: factory.Faker('sentence', nb_words=6, variable_nb_words=True).generate({})[:80]
```
Comment (artcz) [inline conference/tests/factories/talk.py]: line length/wrapping maybe?

**Review by artcz — APPROVED** [review]
lot of changes, LGTM, but as with the other PR – we need a separate test on staging to make sure it works.

File: conference/tests/factories/talk.py (hunk 1 above)
Comment (umgelurgel) [reply conference/tests/factories/talk.py]: Well, the project doesn't have a style guide - I'm aiming to keep the line lengths below 120. If there's a different limit that the changes should adhere to, let me know.

**Comment by fpierfed** [comment]
79?

**Comment by artcz** [comment]
79 sounds good to me as well. :)

**Pull request merged**